apps/ledger-live-mobile/src/newArch/features/Market/screens/MarketList/components/BottomSection/BottomSection.tsx

```diff
@@ -0,0 +1,164 @@
+import { MarketListRequestParams } from "@ledgerhq/live-common/market/types";
+import TrackScreen from "~/analytics/TrackScreen";
+
+const SORT_OPTIONS = {
+  top100: {
+    requestParam: {
+      ids: [],
+      starred: [],
+      orderBy: "market_cap",
+      order: "desc",
+      search: "",
+      liveCompatible: false,
+      sparkline: false,
+      top100: true,
+    },
+    value: "top100",
+  },
+  market_cap_asc: {
+    requestParam: {
+      order: "asc",
+      orderBy: "market_cap",
+      top100: false,
+      limit: 20,
+    },
+    value: "market_cap_asc",
+  },
+  market_cap_desc: {
+    requestParam: {
+      order: "desc",
+      orderBy: "market_cap",
+      top100: false,
+      limit: 20,
+    },
+    value: "market_cap_desc",
+  },
+};
+
+const getIcon = (top100?: boolean, order?: string) =>
+  top100
+    ? IconsLegacy.GraphGrowMedium
+    : order === "asc"
+    ? IconsLegacy.ArrowTopMedium
+    : IconsLegacy.ArrowBottomMedium;
+
+const TIME_RANGES = Object.keys(rangeDataTable)
+  .filter(key => key !== "1h")
+  .map(value => ({
+    requestParam: { range: value },
+    value,
+  }));
+
+interface BottomSectionProps {
+  top100?: boolean;
+  orderBy?: string;
+  order?: string;
+  range?: string;
+  counterCurrency?: string;
+  onFilterChange: (_: MarketListRequestParams) => void;
+  toggleFilterByStarredAccounts: () => void;
+  filterByStarredAccount: boolean;
+}
+
+function BottomSection({
+  top100,
+  orderBy,
+  order,
+  range,
+  counterCurrency,
+  onFilterChange,
+  toggleFilterByStarredAccounts,
+  filterByStarredAccount,
+}: BottomSectionProps) {
+  const navigation = useNavigation();
+  const { t } = useTranslation();
+
+  const timeRanges = TIME_RANGES.map(timeRange => ({
+    ...timeRange,
+    label: t(`market.range.${timeRange.value}`),
+  }));
+  const timeRangeValue = timeRanges.find(({ value }) => value === range);
+
+  const overflowX = ScrollContainerHeader.Header.PADDING_HORIZONTAL;
+
+  return (
+    <ScrollContainer
+      style={{ marginHorizontal: -overflowX, marginTop: 16 }}
+      contentContainerStyle={{ paddingHorizontal: overflowX - 6 }}
+      height={40}
+      horizontal
+      showsHorizontalScrollIndicator={false}
+    >
+      <TrackScreen category="Page" name={"Market"} access={true} />
+      <TouchableOpacity onPress={toggleFilterByStarredAccounts} testID="starred">
+        <StyledBadge bg={filterByStarredAccount ? "primary.c80" : "neutral.c30"}>
+          <Icon
+            name={filterByStarredAccount ? "StarSolid" : "Star"}
+            color={filterByStarredAccount ? "background.main" : "neutral.c100"}
+          />
+        </StyledBadge>
+      </TouchableOpacity>
+      <SortBadge
+        label={t("market.filters.sort")}
+        valueLabel={t(
+          top100 ? `market.filters.order.topGainers` : `market.filters.order.${orderBy}`,
+        )}
+        Icon={getIcon(top100, order)}
+        value={top100 ? "top100" : `${orderBy}_${order}`}
+        options={[
```

Comment:
it seems like the creation of this options array is static and should be moved out to a global var?

Reply:
Not really static as we depend on the translation:
`label: t("market.filters.order.topGainers")`
Static part has been extracted.

Reply:
(if it's not static, it may not be a bad idea to useMemo it, but it's probably a minor problem vs all other thing we can find as improvements)